tests/green_http_test.py

```diff
@@ -1,8 +1,7 @@
 import eventlet
-import six
 import tests
 
-__test__ = six.PY3
+__test__ = True
```

Comment:
I think we could simply remove that line. Pytest would ignore test if `__test__ = False`, so I don't think this line is necessary.

It reflect a time where tests would had to be ignored with python 2.7